dev_suites/dev_demo_ig_stu1/demo_endpoint.rb

```diff
@@ -0,0 +1,25 @@
+module DemoIG_STU1 # rubocop:disable Naming/ClassAndModuleCamelCase
+  class DemoEndpoint < Inferno::DSL::SuiteEndpoint
+    def test_run_identifier
```

Comment:
Why the different style (using `def` instead of how our other dsls look).

Also, should this be called a 'discriminator'?

Reply:
We could call it discriminator, but I don't know that that is more informative. `test_run_discriminator` perhaps.

Making dsl methods for these has questionable benefits but serious downsides.

Pros of not using def:
* You don't have to type `def`
* It looks more like our existing DSL

Pros of using def:
* It's a lot easier to reason about/maintain
* It doesn't mess up inheritance
* You can treat it like a normal ruby class
* Methods are a lot less confusing than procs/lamdas